bot/cogs/stats.py

```diff
@@ -32,6 +36,8 @@ def __init__(self, bot: Bot):
         if CHANNEL_NAME_OVERRIDES.get(message.channel.id):
             reformatted_name = CHANNEL_NAME_OVERRIDES.get(message.channel.id)
 
+        reformatted_name = "".join([char for char in reformatted_name if char in ALLOWED_CHARS])
```

Comment:
List comprehension is not needed.
```suggestion
        reformatted_name = "".join(char for char in reformatted_name if char in ALLOWED_CHARS)
```